packages/react-components/react-menu/src/components/MenuList/useMenuList.ts

```diff
@@ -29,6 +36,27 @@ export const useMenuList_unstable = (props: MenuListProps, ref: React.Ref<HTMLEl
 
   const innerRef = React.useRef<HTMLElement>(null);
 
+  React.useEffect(() => {
+    const element = innerRef.current;
+
+    if (hasMenuContext && targetDocument && element) {
+      const onTabsterMoveFocus = (e: TabsterMoveFocusEvent) => {
+        const nextElement = e.detail.next;
+
+        if (nextElement && element.contains(targetDocument.activeElement) && !element.contains(nextElement)) {
```

Comment:
Is this necessary? all tab presses should just be handled by menu as before

Reply:
This is just an extra safety cushion to make sure that if two menus are mounted at the same time, one menu doesn't unnecessarily prevent events from another.